### ISSUE TITLE
Config Plugin imports from `@expo/config-plugins` instead of `expo/config-plugins`

### ISSUE THREAD

**Issue by wodin** [dooboolab-community/react-native-iap#2749]
https://github.com/dooboolab-community/react-native-iap/blob/e2da9f9b67d659b46aba6a8660c6b47de92ab119/plugin/src/withIAP.ts#L1-L6

Only in Expo SDK 41 to 46 should a config plugin import from `@expo/config-plugins`:
https://github.com/expo/expo/blob/cf6a36e44a179d52aad87bb78e5a511919c59661/docs/pages/config-plugins/development-and-debugging.mdx?plain=1#L67
> For SDK 46 and lower, import the @expo/config-plugins package directly. This is installed automatically by the expo package, but not re-exported as it is in SDK 47 and higher.

In Expo SDK 47 and higher it should import from `expo/config-plugins` (without the `@`).

Also:
https://github.com/dooboolab-community/react-native-iap/blob/e2da9f9b67d659b46aba6a8660c6b47de92ab119/package.json#L121-L123

With the above dependency, `npx expo-doctor` complains about the version of `@expo/config-plugins` in an Expo SDK 51 project:

```
Expected package @expo/config-plugins@~8.0.0
Found invalid:
  @expo/config-plugins@7.9.2
  (for more info, run: npm why @expo/config-plugins)
Advice: Upgrade dependencies that are using the invalid package versions0.
```

You should probably update the dependencies/peerDependencies as per this instead of depending on `@expo/config-plugins`:
https://docs.expo.dev/config-plugins/development-and-debugging/#install-dependencies

**Comment by hyochan** [comment]
Looks like resolved in #2758 which is applied in `12.14.0`.
Feel free to re-open when it didn't.

**Comment by PrinceD96** [comment]
@hyochan both `12.14.0` and main branch still import from `@expo/config-plugins`

I think we should re-open

**Comment by hyochan** [comment]
Apologies! I initially thought #2758 could be a fix.
However, upon reconsideration, I think this shouldn't be included in the library's dependencies. Instead, it should be installed manually in local projects. I'll update the import statement accordingly.

**Comment by hyochan** [comment]
> @hyochan both `12.14.0` and main branch still import from `@expo/config-plugins`
> 
> I think we should re-open

Fixed in [12.14.1](https://github.com/dooboolab-community/react-native-iap/releases/tag/12.14.1)
